src/Findologic/Response/Xml21ResponseParser.php

```diff
@@ -146,7 +149,7 @@ private function buildCategoryQueryInfoMessage(array $params): CategoryInfoMessa
         $categoryInfoMessage = QueryInfoMessage::buildInstance(
             QueryInfoMessage::TYPE_CATEGORY,
             null,
-            $filters['cat']->getDisplay(),
+            isset($filters['cat']) ? $filters['cat']->getDisplay() : 'Category',
```

Comment:
The default should come from the Smart Suggest blocks. Have a look at how I've implemented it for Shopware 5 with Findologic-API:

https://github.com/findologic/plugin-shopware-5/blob/5cd716d7029c2e2d9c53224a9f1c4bf0381ade5e/FinSearchUnified/Bundle/SearchBundleFindologic/ResponseParser/Xml21ResponseParser.php#L203.